### ISSUE TITLE
Introduce abstract class to implement all content behaviours

### ISSUE THREAD

**Issue by alexander-schranz** [sulu/SuluContentBundle#112]
Currently if you want to implement all content behaviours for your entit you need todo the following:

```
class ExampleContentProjection implements ContentProjectionInterface, SeoInterface, ExcerptInterface, TemplateInterface, WorkflowInterface
{
    use SeoTrait;
    use ExcerptTrait;
    use TemplateTrait;
    use WorkflowTrait;
    use WorkflowTrait;
```

To avoid this we should provide a Abstractclass implementing all this interfaces and traits for better developer experience.

**Comment by nnatter** [comment]
I dont think it is a big burden to explicitly use the traits in your custom entity, but an abstract class would definitely make things easier for new developers. 

but how should we call these classes? not sure if `AbstractDimensionContent` is a good idea because it does not indicate that it implements all the behaviours 🤔

**Comment by alexander-schranz** [comment]
@nnatter `AbstractContentfulEntity` 🙈

**Comment by nnatter** [comment]
that one sounds very similar to `ContentRichtEntity` 🙈
